lib/ask/runtime/flow.ex

```diff
@@ -81,7 +81,11 @@ defmodule Ask.Runtime.Flow do
 
     case skip_logic do
       nil ->
-        flow.current_step + 1
+        if (flow.current_step == flow.questionnaire.steps) do
```

Comment:
`flow.current_step` is an integer. `flow.questionnaire.steps` is an array. So this comparison is always false and the code executed is the same that existed before the change. Please, revert this change.